SonarQube.Bootstrapper/ArgumentProcessor.cs

```diff
@@ -29,9 +29,11 @@ public static class ArgumentProcessor
 
         private const string BeginId = "begin.id";
         private const string EndId = "end.id";
+        private const string InstallTargetsId = "installTargets.id";
 
         public const string BeginVerb = "begin";
         public const string EndVerb = "end";
+        public const string InstallTargetsVerb = "/i:";
```

Comment:
"/install:" might be better, particularly since we don't have any command line help for the time being.